docs/insomnia/migrate-from-designer.md

```diff
@@ -35,28 +39,44 @@ If there is an unexpected error during migration, you will be shown the error th
 
-The <app-data-dir>/insomnia/core-backup directory contains the following data:
+See [Backup](#backup) for a list of `app-data-dir` locations based on your OS.
 
-* insomnia.{model}.db database files
+The `<app-data-dir>/insomnia/core-backup` directory contains the following data:
+
+* `insomnia.{model}.db` database files
 * plugins folder
 * responses folder
 * version-control folder
 
-Each of these items also exists in the parent folder, <app-data-dir>/insomnia, which contains the "current" data used by the application. 
+Each of these items also exists in the parent folder, `<app-data-dir>/insomnia`, which contains the current data used by the application.
 
-In order to manually restore:
+In order to manually restore, do the following:
 
 1. Close Insomnia
-2. Delete the plugins, responses, and version-control folders in the <app-data-dir>/insomnia live data directory
-3. Copy-paste everything from <app-data-dir>/insomnia/core-backup into <app-data-dir>/insomnia, overwriting everything that exists
+2. Delete the plugins, responses, and version-control folders in the `<app-data-dir>/insomnia` live data directory
+3. Copy-paste everything from `<app-data-dir>/insomnia/core-backup` into `<app-data-dir>/insomnia`, overwriting everything that exists
 4. Launch Insomnia
 
-## Edge cases
+## Edge case
+
+If you moved your workspace from Core to Designer several months ago, and are now merging the same workspace (with the same id) from Designer back into Core, this edge case applies. The merge won't remove any children of the workspace (such as requests, request groups, certificates), but if a duplicate id is found, it will favor the data coming from Designer.
+
+In the unlikely scenario that you can't see your requests or workspace in Insomnia after migration, [restore your data from your backup](#restore-from-backup). Alternatively, because the migration is primarily additive and will not remove anything from the existing database (it will overwrite if the same id is found), some entries may become orphaned. Find orphaned entries by opening the `insomnia.{model}.db` files.
+
+If you have orphaned entries, or are not seeing data you expect to see when you launch Insomnia, send an email to [Support](mailto:support@insomnia.rest) outlining your steps, including your application logs, and the team will assist where possible.
+
+## Migration Alternatives
+
+You can still migrate your data in the following situations:
 
-There are potential edge cases where a user moved their workspace from Core to Designer several months ago, and is now merging the same workspace (with the same id) from Designer back into Core. This merge will not remove any children of the workspace (eg. requests, request groups, certificates, etc), but if a duplicate id is found it will favor the data coming from Designer.
+* You've never installed Insomnia but I have Insomnia Designer
```

Comment:
is the change of person from 2nd person to 1st person intentional?

Reply:
Oops, nope! Good catch!

Reply:
Took out "I". Resolved.